src/app/home-page/widget/entry-box/entry-box.component.scss

```diff
@@ -0,0 +1,46 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.entry-boxes {
```

Comment:
Not sure if we should be having all this component-specific CSS. @kathy-t will know better than I.

Reply:
Hmm, it seems component specific enough because it's a lot of layout styling